README.md

```diff
@@ -64,3 +64,105 @@ $ docker run --volume $PWD:/home/harambe/project --volume ~/.tokenlists:/home/ha
 This project is in development and should be considered a beta.
 Things might not be in their final state and breaking changes may occur.
 Comments, questions, criticisms and pull requests are welcomed.
+
+## Full Environment Execution Example
+
+Running the `Quick Usage` and `Docker Usage` examples will fail if you do not have a full environment setup.
```

Comment:
```suggestion
Running the `Quick Usage` and `Docker Usage` with the provided example will fail if you do not have a fully-configured environment.
```

Reply:
And then add another sentence like "most common issues when using the SDK rely on proper configuration of Ape plugins to unlock the behavior you desire, so make sure that your docker builds are configured correctly" and then a link to Ape docs on plugin installation/config